mine.js

```diff
@@ -11,14 +11,20 @@ const path = require('path')
 const fs = require('fs')
 const spawn = require('child_process').spawn
 const IPFS = require('./lib/ipfs')
+const geth = require('./lib/geth')
 
 function checkForModels (mineAddress, contractAddress, web3) {
-  IPFS.connect()
-  .then(ipfs => {
-    console.log(`💾  Connected to IPFS. Online:`, ipfs.isOnline())
-    trainModels(mineAddress, contractAddress, web3, ipfs)
+  geth.connect(mineAddress)
+  .then(() => {
+    console.log(`📄  Connected to Geth`)
+    IPFS.connect()
```

Comment:
Use promise chaining by returning the promises instead of building pyramids of doom

https://javascript.info/promise-chaining#returning-promises

Reply:
👍 "pyramids of doom" lol